src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,11 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        for (int i = 0; i < 3; i++) {
+            Lottery randomBall = new Lottery();
+            String message = "" + randomBall.getRandomBall();
+            System.out.println(message);
```

Comment:
```suggestion
            Ball ball = randomBall.getRandomBall();
            System.out.println(ball);
```

метод System.out.println под капотом вызывает метод toString на объексте класса, который мы передаем на вход

Reply:
Убрал создание переменной типа String, выводим на экран сразу ball